src/OpenTelemetry/Metrics/OpenTelemetryMetrics.cs

```diff
@@ -0,0 +1,65 @@
+﻿// <copyright file="OpenTelemetryMetrics.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using OpenTelemetry.Metrics.Export;
+using static OpenTelemetry.Metrics.MeterProviderSdk;
+
+namespace OpenTelemetry.Metrics
+{
+    /// <summary>
+    /// OpenTelemetry helper.
+    /// </summary>
+    public class OpenTelemetryMetrics
```

Comment:
Not sure of better names.. If this is called OpenTelemetryMetrics, then `OpenTelemetrySdk` should be called `OpenTelemetryTrace`

https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/src/OpenTelemetry/Trace/OpenTelemetrySdk.cs

Or should i name to OpenTelemetryMetricHelper and TracerHelper?

Or MetricsHelper, TracerHelper?

or Metrics, Traces

or Cijo?

Reply:
Flamebait :fire:

Reply:
Do we need this class at all? Could we move it to MeterProvider?
```csharp
public class MeterProvider
{
   public static MeterProvider Create(Action<MeterBuilder> configure)) {...}
}
```

Reply:
MeterProvider is in API project only.

Reply:
I wouldn't use OT, since we know that everything is OT. So, `MetricHelper` is good, I guess

Reply:
Extension method?